pkg/cloud/services/loadbalancer/loadbalancer.go

```diff
@@ -268,7 +267,6 @@ func (s *Service) ReconcileLoadBalancerMember(clusterName string, machine *clust
 			Name:         name,
 			ProtocolPort: port,
 			Address:      ip,
```

Comment:
So just that I get it right. Instead of setting the correct subnet (which is invisible to Octavia) we don't set it but Octavia then chooses to subnet automatically? (and I guess it chooses the one we would set)

(fyi, OpenStack cloud controller manager also sets the subnet id, but only if it's configured in the cloud.conf)

Reply:
Yes. If you omit the subnet_id parameter, the vip_subnet_id for the parent load balancer will be used for the member subnet UUID.
https://docs.openstack.org/api-ref/load-balancer/v2/?expanded=create-member-detail#create-member

Reply:
Sounds good, thx for the info